sensing/pointcloud_preprocessor/include/pointcloud_preprocessor/crop_box_filter/crop_box_filter_nodelet.hpp

```diff
@@ -92,7 +83,7 @@ class CropBoxFilterComponent : public pointcloud_preprocessor::Filter
   rcl_interfaces::msg::SetParametersResult paramCallback(const std::vector<rclcpp::Parameter> & p);
 
 public:
-  PCL_MAKE_ALIGNED_OPERATOR_NEW
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
I think this can't be changed due to Rolling/Humble support.
https://github.com/autowarefoundation/autoware.universe/pull/804/files#r860083464